hypercompose/requests_aws4auth/aws4auth.py

```diff
@@ -186,7 +187,7 @@ class AWS4Auth(AuthBase):
                             supplied directly on the command line
 
     """
-    default_include_headers = ['host', 'content-type', 'date', 'x-hyper-*']
+    default_include_headers = ['content-type', 'host', 'content-type', 'date', 'x-hyper-*']
```

Comment:
I think the overall problem was indeed to provide a content-type header in the request.
Anyway. this works, and allow me to move forward. Eventually I will contribute to the aws4auth project in order to put those aws strings as constant parameters, so this code is just PoC